lexers/testdata/powershell.expected

```diff
@@ -12,26 +12,311 @@
   {"type":"Name","value":"-Name"},
   {"type":"Text","value":" "},
   {"type":"Name","value":"-Path"},
-  {"type":"Text","value":" "},
+  {"type":"Text","value":"  "},
   {"type":"Name","value":"C:"},
   {"type":"Punctuation","value":"\\"},
   {"type":"Text","value":" "},
   {"type":"Punctuation","value":"*."},
   {"type":"Name","value":"txt"},
+  {"type":"Text","value":"\n\n"},
```

Comment:
Should this be `TextWhitespace`?

Reply:
Very minor nit and won't make any difference, so I merged it.

Reply:
I might do more improvements a bit later, so I can look into this next time